tests/agents/system/test_disk_fill.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright 2021, Yahoo
+#  Licensed under the terms of the Apache 2.0 license. See the LICENSE file in the project root for terms
+import math
+import os
+import shutil
+from unittest import TestCase
+
+from mockito import unstub, when
+
+from ychaos.agents.agent import AgentState
+from ychaos.agents.system.diskFill import DiskFill, DiskFillConfig
+
+
+class TestDiskFill(TestCase):
+    def setUp(self) -> None:
+        pass
+
+    def test_disk_fill_updates_state_when_complete(self):
+        disk_fill_config = DiskFillConfig(duration=0.1, partition="./", partition_pct=5)
+
+        stat = shutil.disk_usage(path=os.getcwd())
```

Comment:
https://github.com/yahoo/ychaos/blob/8eadeaed7da8370228bbd8bd345a26df81e6f7b0/docs/contribution.md#L128-L130

```suggestion
        stat = shutil.disk_usage(path=str(Path().resolve()))
```